.github/workflows/auto-update-collector-version

```diff
@@ -0,0 +1,32 @@
+name: Auto-update Collector Version
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 * * * *'
+
+jobs:
+  build:
+    name: Auto-update on new collector version
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.18
+
+      - name: Auto-update
+        run: ./scripts/update-collector-version.sh
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GIT_AUTHOR_NAME: Auto-update CI workflow
+          GIT_COMMITTER_NAME: Auto-update CI workflow
+
+          # Set svrnm's email, otherwise CLA checks will fail
+          GIT_AUTHOR_EMAIL: neumanns@cisco.com
+          GIT_COMMITTER_EMAIL: neumanns@cisco.com
```

Comment:
We should get an official OTel bot account for this. WDYT @austinlparker @cartermp?

Reply:
Here I just copied from @jpkrohling's work, it looks like that there's some issues with CLA, not sure if we can work around that easily?

Reply:
see discussion in https://github.com/open-telemetry/community/issues/809 and https://github.com/open-telemetry/community/issues/954

TLDR, there have been no objections to using the @opentelemetrybot account which has signed the CLA. It's on me to officially add this to the community repository assets page, I'll send a PR to the community repo later today.

```
git config user.name opentelemetrybot
git config user.email 107717825+opentelemetrybot@users.noreply.github.com
```

Reply:
sent community PR to (hopefully) make this approach official: https://github.com/open-telemetry/community/pull/1285